convert2rhel/pkgmanager/__init__.py

```diff
@@ -65,10 +65,10 @@ def create_transaction_handler():
         from convert2rhel.pkgmanager.handlers.yum import YumTransactionHandler
 
         return YumTransactionHandler()
-    elif TYPE == "dnf":
-        from convert2rhel.pkgmanager.handlers.dnf import DnfTransactionHandler
 
-        return DnfTransactionHandler()
+    from convert2rhel.pkgmanager.handlers.dnf import DnfTransactionHandler
+
+    return DnfTransactionHandler()
```

Comment:
Just a note, since you were asking about how to rewrite code that requires either dnf or yum in one of our 1::1s.  This can be done at the toplevel  with something like:
``` python
try:
    from yum import *
    # [...]
    from convert2rhel.pkgmanager.handlers.yum import YumTransactionHandler as TransactionHandler
except ImportError:
    from convert2rhel.pkgmanager.handlers.dnf import DnfTransactionHandler as TransactionHandler
```

The naming (`TransactionHandler vs DnTransactionHandler) could be done inside of the specific handlers instead of in this file as well.

(This should not be done in this PR.  Just seeing this reminded me of our discussion.)

Reply:
I think that it would cause a recursive import, as the `{Yum,Dnf}TransactionHandler` import from pkgmanager to do a lot of operations. 

But once we have figured up a good way of transitioning to the refactor of yum/dnf, we can take care of that.